EnglishFilter/sections/antiadblock.txt

```diff
@@ -433,6 +433,8 @@ gelbooru.com,thegatewaypundit.com,flagandcross.com#@#.text_ads
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38961
 @@||solve-all-problems.info/googleads.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/38753
+@@||egyanime.com/layout/js/ads.js
```

Comment:
Site is not English.

Reply:
I think we can stop devide rules in `foreign.txt` for the langs and add like another rules, because it is not English filter, but Base:)

Reply:
asked in chat, approved for this time